pkg/volume/util.go

```diff
@@ -38,6 +38,13 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
+const (
+	// GiB is used by PV/PVC. 1 GiB = 1024 * 1024 * 1024 Bytes
+	VolumeSizeGiB = 1024 * 1024 * 1024
```

Comment:
are we using this variable anywhere?

Reply:
Yeah. A lot of places. Such as [this line](https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/flocker/flocker_util.go#L76). If this bug fix is accepted, I am willing to submit another pr to fix [this issue](https://github.com/kubernetes/kubernetes/issues/56225) with @gnufied.

Reply:
the line you linked is the one you changed to "volume.VolumeSizeGB." So I dont think the GiB constant is being used anywhere anymore in the current version of your PR.